_assets/scss/modules/_thumbnail.scss

```diff
@@ -59,8 +60,14 @@
     display: block;
   }
 
-  .thumb { // image margin bottom
+  /*.thumb { // image margin bottom — replaced by .h4 margin-top
     margin-bottom: 7px;
+  }*/
+
+  .h4 {
+    position: absolute; 
+    line-height: 1em;
```

Comment:
Lets remove the line-height and adjust the margin accordingly. That way if for any reason the text ever wraps in the future it will still look as the font was intended. I would rather change that at the font leve.